File: Bubble-Sort/js/simple.js
const arr = []

function initData(count){
    for(let i = 0 ; i < count ; i++){
        arr.push(Math.floor(Math.random()*50-20))
    }
}

initData(10)

console.log(arr)

function sort(){
    for(let i = 0 ; i < arr.length ; i++){
        for(let j = 1 ; j < arr.length - i ; j++){
            if(arr[j-1] > arr[j]){
                const tmp = arr[j-1]
                arr[j-1] = arr[j]
                arr[j] = tmp
            }
        }

    } 
}

sort()

console.log(arr)

File: Bubble-Sort/python/simple.py
import random;

arr = [random.randint(-20,50) for _ in range(10)]

print(arr)

def sort():
    for i in range(0,len(arr)):
        for j in range(1,len(arr)-i):
            if arr[j-1] > arr[j] :
                arr[j-1],arr[j] = arr[j],arr[j-1]
                

sort()
print("sorted:",arr)